extra/cba_settings_userconfig/cba_settings.sqf

```diff
@@ -16,12 +16,20 @@ force ace_switchunits_enableSwitchUnits = false;
 force ace_vehiclelock_vehicleStartingLockState = -1;
 force ace_scopes_enabled = true;
 force ace_noradio_enabled = true;
-force force ace_advanced_fatigue_enabled = false;
 force force ace_microdagr_mapDataAvailable = 2;
 force force ace_gforces_enabledFor = 1;
 force force ace_cargo_enable = true;
 force ace_cargo_paradropTimeCoefficent = 2;
 
+// ACE Advanced Fatigue
+force ace_advanced_fatigue_enabled = false;
+force ace_advanced_fatigue_enableStaminaBar = true;
```

Comment:
Setting soft enforce of default values is kind of useless

Reply:
we do so with some already.

Reply:
no need for more

Reply:
there